app/views/layouts/application.html.erb

```diff
@@ -28,7 +28,7 @@
     <%= render 'shared/testing' %>
     <%= render 'shared/tracking' %>
 
-    <%= javascript_include_tag "https://maps.googleapis.com/maps/api/js?key=#{ Credential.fetch(:google, :maps_api_key) }" %>
+    <%= javascript_include_tag "https://maps.googleapis.com/maps/api/js?key=#{ Credential.fetch(:google, :maps_api_key) }&libraries=places" %>
```

Comment:
Did you test this?... it needs to be `html_safe` or it is escaping the `&libraries=places`